src/translations/en_US.js

```diff
@@ -24,6 +24,11 @@ module.exports = {
   paypalAccountTokenizationFailedError: 'Something went wrong adding the PayPal account. Please try again.',
   paypalFlowFailedError: 'Something went wrong connecting to PayPal. Please try again.',
   paypalTokenizationRequestActiveError: 'PayPal payment authorization is already in progress.',
+  applePayAccountTokenizationFailedError: 'Something went wrong with Apple Pay. Please try again.',
+  applePayActiveCardError: 'Add a supported card to your Apple Pay wallet.',
+  applePayBrowserNotSupported: 'Browser does not support Apple Pay.',
```

Comment:
Do we want to show this error or hide Apple Pay as an option?

Reply:
I think we just want it to not be shown. No need to have an error.

Reply:
This can be removed.